guides/v2.2/cloud/project/profile-database-queries.md

```diff
@@ -0,0 +1,69 @@
+---
+group: cloud-guide
+title: Profile database queries
+functional_areas:
+  - Cloud
+  - Setup
+  - Services
+---
+
+When making changes to a database, you may find it helpful to run profiling to identify the impact of those changes. Profiling maintains a log of database queries and increases run time precision.
+
+To enable database query profiling:
```

Comment:
This is low importance, but most of cloud uses H4 for a procedure title. We don't have a class in the CSS yet. 😖

Reply:
i thought i might be able to sneak this past you ;)

Reply:
🤣